packages/apex-node/src/streaming/streamingClient.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { Client as FayeClient } from 'faye';
+import { ApexFayeClient } from '../../typings/modules/faye';
```

Comment:
turns out this was no bueno b/c the typings isn't included in the build so it can't be imported this way.  See later commit comments for my current work around.